src/main/java/com/example/petbuddybackend/service/chat/ChatService.java

```diff
@@ -60,4 +218,12 @@ private ChatMessageDTO convertTimeZone(ChatMessageDTO message, ZoneId timeZone)
         message.setCreatedAt(zonedDateTime);
         return message;
     }
+
+    private ChatRoomDTO convertTimeZone(ChatRoomDTO chatRoom, ZoneId timeZone) {
```

Comment:
zamiast tego w mapperze mozna tak:
```suggestion
        @Mapping(target = "senderEmail", source = "sender.email")
    @Mapping(target = "chatId", source = "chatRoom.id")
    @Mapping(target = "createdAt", source = "createdAt", qualifiedByName = "mapToZonedDateTime")
    ChatMessageDTO mapToChatMessageDTO(ChatMessage chatMessage, @Context ZoneId zoneId);
    
    @Named("mapToZonedDateTime")
    default ZonedDateTime mapToZonedDateTime(ZonedDateTime date, @Context ZoneId zoneId) {
        return date.withZoneSameInstant(zoneId);
    }
```

Reply:
O fajne rozwiązanie

Reply:
tez mnie sie podoba